### PR TITLE
Dashboard: fix elective tags

### LINKED ISSUE #4484: Tags for elective courses are not visible
### Steps to Reproduce
In a program with electives courses the courses are labeled with "Core" or "Elective" tag.

### Expected Behavior
I don't remember how the ui looked like, but the tags should be visible.

### Actual Behavior

Screen Shot 2019-12-18 at 1 08 28 PM

### PR DESCRIPTION
**Pull request by annagav**
#### Pre-Flight checklist

- [ ] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
  - [ ] Mobile width screenshots
  - [x] Tag @ferdi or @pdpinch for review
- [ ] Migrations
  - [ ] Migration is backwards-compatible with current production code
- [ ] Testing
  - [ ] Code is tested
  - [ ] Changes have been manually tested
- [ ] Settings
  - [ ] New settings are documented and present in `app.json`
  - [ ] New settings have reasonable development defaults, if applicable

#### What are the relevant tickets?
Fix #4484

#### What's this PR do?
Add the missing styles to the elective tags.

#### How should this be manually tested?
Test with a program that has elective courses
In the dashboard, the list of courses should be labeled accordingly:
Screen Shot 2019-12-18 at 1 24 58 PM

### CODE REVIEW

**Comment by annagav** [comment]
Before: 
Screen Shot 2019-12-18 at 1 28 41 PM


After:
Screen Shot 2019-12-18 at 1 29 00 PM

File: static/js/components/dashboard/CourseRow.js [hunk 1]
```diff
@@ -206,11 +206,8 @@ export default class CourseRow extends React.Component {
   }
   getCourseTag = (): React$Element<*> => {
     const { course } = this.props
-    return (
-      <div className="elective-tag">
-        {course.is_elective ? "Elective" : "Core"}
-      </div>
-    )
+    const tag = course.is_elective ? "Elective" : "Core"
+    return <div className={`elective-tag ${tag}`}>{tag}</div>
```
Comment (noisecapella) [inline static/js/components/dashboard/CourseRow.js]: Do we need the `tag` in the `className` here? We should remove it if not. It doesn't look like there is a CSS class for these names, and if so it should not contain capital letters.

Comment (annagav) [reply static/js/components/dashboard/CourseRow.js]: yes, there is a class that corresponds to the tag name.

**Comment by annagav** [comment]
@noisecapella I changed the class to lower case, and made tag all uppercase, because I think it looks better like that.
This PR effects 3 different places:
1) The CourseRow on the dashboard
2)The courses list on the programs page
3) Program record view

**Pull request merged**